src/rsa.rs

```diff
@@ -163,8 +164,133 @@ fn parse_public_key<'a>(input: untrusted::Input<'a>) ->
     })
 }
 
+#[allow(non_camel_case_types)]
+enum BIGNUM {}
+#[allow(non_camel_case_types)]
+enum BN_MONT_CTX {}
+
+#[allow(non_camel_case_types)]
+#[repr(C)]
+/// Needs to be kept in sync with `struct rsa_st` (in `include/openssl/rsa.h`).
```

Comment:
NIT: I think the comment should go above the `#[...]` stuff, unless there's some Rust style guide that says the opposite.